ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WorkloadCveViewContext.ts

```diff
@@ -1,10 +1,27 @@
 import { createContext } from 'react';
 import { NonEmptyArray } from 'utils/type.utils';
 
+import type { VulnerabilityState } from 'types/cve.proto';
 import { QuerySearchFilter, WorkloadEntityTab } from '../types';
 
 export type WorkloadCveView = {
-    getAbsoluteUrl: (path: string) => string;
+    urlBuilder: {
```

Comment:
> cleaning up some verbosity at the link creation site

Yes, consistent argument order and difference in argument types seems realistic to scale into the future,